### ISSUE TITLE
[BITV] 9.4.1.2/3.4 - The "Details" section contains a favorite button, where the status is only communicated to screen reader users in the "on" state using the aria-pressed="true" attribute.  (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41892]
The "aria-pressed" attribute should be used in both states.

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n4

9.4.1.2/11.4 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n4

**Comment by susnux** [comment]
probably already fixed with #41450 or #41481

**Comment by emoral435** [comment]
this seems relatively easy to change if its not fixed with either of those pr's, but ill check it when I am home currently at university so just going to assign myself to this issue to either code / close this when I get home)

**Comment by giantmolecules1** [comment]
Nice

**Comment by emoral435** [comment]
After doing some investigating, it seems like this button does not have any aria-pressed labelled associated to it (correct me if I am wrong!), so it will need to be added to the actual button itself alongside its own toggle functionality.

**Comment by emoral435** [comment]
Also seems like this is a problem due to the NcAppSidebar vue component -

**Comment by JuliaKirschenheuter** [comment]
This issue can be closed because not relevant anymore after https://github.com/nextcloud/server/pull/41481. In other places it is just a button and no improvements have to be done.
